ipa-core/src/ff/ec_prime_field.rs

```diff
@@ -219,6 +221,14 @@ impl Field for Fp25519 {
     const ONE: Fp25519 = Fp25519::ONE;
 }
 
+impl ExtendableField for Fp25519 {
```

Comment:
`ExtendableField` is automatically implemented for `PrimeField`, but `PrimeField` assumes less than 128 bits, which wouldn't work here.

Reply:
yes PrimeField stores the prime in a u128